tools/distrib/python/docgen.py

```diff
@@ -32,12 +32,10 @@
     help='GRPC/GPR libraries build configuration',
     default='opt')
 parser.add_argument('--submit', action='store_true')
-parser.add_argument('--gh-user', type=str, help='GitHub user to push as.')
 parser.add_argument(
-    '--gh-repo-owner',
+    '--repo-owner',
```

Comment:
Why the name change on this flag? Do we know who all is automating against this script?